asm/bss_8001C4D0_1.s

```diff
@@ -35,19 +35,19 @@ glabel D_8018EB34
 glabel D_8018EB38
 .skip 64
 
-glabel D_8018EB78
+glabel gControllerPak1NumFilesUsed
 .skip 4
 
-glabel D_8018EB7C
+glabel gControllerPak1MaxWriteableFiles
 .skip 4
 
-glabel gControllerPakNumPagesFree
+glabel gControllerPak1NumPagesFree
 .skip 4
 
-glabel D_8018EB84
+glabel gControllerPak1FileNo
```

Comment:
Is this like the currently used file?

Try to avoid abbreviations. gControllerPak1CurrentFile?

Reply:
yes, it also specifies the pak number because mk64 run code for multiple paks for some reason

Reply:
well, more like a file descriptor